Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.h

```diff
@@ -71,6 +71,7 @@ class CDMInstanceOpenCDM final : public CDMInstance {
 
     // FIXME: Session handling needs a lot of love here.
     String getCurrentSessionId() const;
+    String getSessionIdByInitData(const Ref<SharedBuffer>&) const;
```

Comment:
`sessionIdByInitData`

Reply:
Why we need to change the name. We made this naming uniform with similar function in the CDMOpenCDM getCurrentSessionId, getKeyStatus, getSessionLoadStatus etc. If it is really required we can change it

Reply:
Yes, it is. Those other names are not right either. I might have been a lousy reviewer before. Sorry for that.

Reply:
updated